faster_whisper/utils.py

```diff
@@ -21,6 +21,7 @@
     "large-v1": "guillaumekln/faster-whisper-large-v1",
     "large-v2": "guillaumekln/faster-whisper-large-v2",
     "large": "guillaumekln/faster-whisper-large-v2",
+    "large-v3": "bababababooey/faster-whisper-large-v3",
```

Comment:
who is owning that ?
Can systran create its hub to keep ownership of the models and easily update old models and upload new models ?

Reply:
> who is owning that ?

Some user posted that link on issues.

Best would be to keep model under official acc like "systran", maybe move all models there.

Reply:
Good point, we are waiting for the release on CTranslate2, then push the new converted model (with the fix by https://github.com/OpenNMT/CTranslate2/pull/1546) to Systran organization

Reply:
@nguyendc-systran thank you! So essentially just waiting for @stillmatic to do the last fixes you mentioned before merge?

Reply:
@flyingleafe 
Why tokenizer.json & vocabulary.json is not he same in https://huggingface.co/flyingleafe/faster-whisper-large-v3 vs your https://huggingface.co/flyingleafe/faster-whisper-large-v3 ?

Reply:
@Purfview I think you accidentally pasted the same URL.

Reply:
@AvivSham Fixed it.

Reply:
Not sure if I missed out any but in tokenizer.json, there is a difference in token 50363, nospeech vs nocaptions.

Reply:
IMHO, yes.
Another point may be interesting / relevant is about this benchmark: https://github.com/SYSTRAN/faster-whisper/pull/548#issuecomment-1814884183.   
Not sure if @funboarder13920 had a chance for that?

Reply:
Yep, the inference between openai/hf/faster_whisper are not exactly the same but very similar. I guess I was witnessing the differences between v2 and v3.